### ISSUE TITLE
Way of retrieving the last stored value for a metric

### ISSUE THREAD

**Issue by ghost** [graphite-project/graphite-web#1170]
I've tried to use carbon to store some custom metrics retrieved via SNMP, then I want to display the last added value in a separate application, however I couldn't find a simple way to retrieve only the value of the last added sample, specifying the last time/date doesn't give me a round value (64b integer counters) but it returns a floating point value which I'm 100% sure does not correspond to that specific sample.
So in short is there a way of saying to ceres/carbon, please give me only the last stored value for this metric?
